Exec/Make.auto_source

```diff
@@ -47,8 +47,12 @@ CEXE_headers += prob_parameters.H
 $(CASTRO_AUTO_SOURCE_DIR)/prob_parameters.cpp: $(CASTRO_AUTO_SOURCE_DIR)/prob_params_auto.F90
 
 PROB_PARAMS_TEMPLATE := $(TOP)/Util/scripts/prob_params.template
-PROB_PARAMETERS := $(PROBLEM_DIR)/_prob_params
-DEFAULT_PROB_PARAMETERS := $(TOP)/Source/problems/_default_prob_params
+# search Bdirs for _prob_params files
+PROB_SEARCH := $(foreach dir, $(Bdirs), $(TOP)/$(dir))
+PROB_PARAMETERS := $(shell $(TOP)/Util/scripts/findparams.py $(PROB_SEARCH))
```

Comment:
Can't you also do this in GNU make with a wildcard on $(Bdirs)/_prob_params, instead of writing a new script?

Reply:
ah indeed, I think maybe that could work.  Let me try.

Reply:
okay, that's implemented.  Sedov compiles okay with the changes